modules/local/megahit/interleaved/main.nf

```diff
@@ -33,6 +33,7 @@ process MEGAHIT_INTERLEAVED {
         $pair_ends \\
         ${single_ends} \\
         -t $task.cpus \\
+        -m ${task.memory.toBytes()} \\
         $args \\
         --out-prefix $assembly
 
```

Comment:
stub section is missing in this script.

Reply:
it does miss in many scripts. are you suggesting to add here for a specific reason?

Reply:
My stub run failed at this step, which is why I suggested making the change here. It's possible that other modules might also require updates to include a stub section. Is the stub working for you when you run the pipeline locally?

Reply:
gotcha! Now I fixed it and -stub-run works with test profile.